phoenix-core/src/it/java/org/apache/phoenix/end2end/BasePermissionsIT.java

```diff
@@ -142,37 +148,45 @@ void startNewMiniCluster(Configuration overrideConf) throws Exception{
 
         Configuration config = testUtil.getConfiguration();
         enablePhoenixHBaseAuthorization(config);
-        configureNamespacesOnServer(config);
+        configureNamespacesOnServer(config, isNamespaceMapped);
         config.setBoolean(LocalHBaseCluster.ASSIGN_RANDOM_PORTS, true);
-        if (overrideConf != null) {
-            config.addResource(overrideConf);
-        }
 
         testUtil.startMiniCluster(1);
-        initializeUsers(testUtil.getConfiguration());
+        superUser1 = User.createUserForTesting(config, SUPERUSER, new String[0]);
+        superUser2 = User.createUserForTesting(config, "superUser2", new String[0]);
     }
 
-    private void initializeUsers(Configuration configuration) {
-
-        superUser1 = User.createUserForTesting(configuration, SUPERUSER, new String[0]);
-        superUser2 = User.createUserForTesting(configuration, "superUser2", new String[0]);
-
-        regularUser1 = User.createUserForTesting(configuration, "regularUser1", new String[0]);
-        regularUser2 = User.createUserForTesting(configuration, "regularUser2", new String[0]);
-        regularUser3 = User.createUserForTesting(configuration, "regularUser3", new String[0]);
-        regularUser4 = User.createUserForTesting(configuration, "regularUser4", new String[0]);
-
-        groupUser = User.createUserForTesting(testUtil.getConfiguration(), "groupUser", new String[] {GROUP_SYSTEM_ACCESS});
-
-        unprivilegedUser = User.createUserForTesting(configuration, "unprivilegedUser", new String[0]);
+    @Before
+    public void setUp() {
```

Comment:
I would move this right after the constructor for better readability.

Reply:
doSetup is called even before the setup happens, is that correct? in that case ignore the comment.

Reply:
doSetup is called once per class, while setup is called before each test executes. I'll rename these.